### ISSUE TITLE
no html or text content with multipart/*

### ISSUE THREAD

**Issue by markusramsak** [zbateson/mail-mime-parser#115]
```
        $mailParser = new MailMimeParser();
        $message = $mailParser->parse($rawMessage); // file_get_contents('eml.txt')
        $htmlText = $message->getHtmlContent(); // null
        $plainText = $message->getTextContent(); // null
```

should be plain text content and one pdf attachment

Here is the original .eml-File to download:
[eml.txt](https://github.com/zbateson/mail-mime-parser/files/4678227/eml.txt)

**Comment by zbateson** [comment]
@markusramsak any idea what may be causing that, whether it's a problem in the email or the library, if you've tried anything to remedy it, version of library you're using, anything else that could help?

**Comment by markusramsak** [comment]
I am using the latest version of your library and I think it is simply a new kind of mail content. Even the preview on mac os displays an empty content. Only if I open the .eml-File with the Mail.app itself it displays the text content and the pdf attachment.

therefore I assume it is a problem with the handling of the library of the content type "Content-Type: multipart/*; charset="ISO-8859-15";"

**Comment by zbateson** [comment]
Thanks for the info -- I must say I've never heard of/seen multipart/*, do you know what mail program created that content type, and if it's common?

**Comment by markusramsak** [comment]
I have parsed more than 20000 mails with your library and this one happens for with content type multipart/* happens for the first time. Therefore I think it is not common. It is a mail after a purchase from https://www.fischersports.com – they have made a mistake with generating a correct mail.

change multipart/* with multipart/mixed and it works :-)

**Comment by zbateson** [comment]
Weird, I wonder where they got that idea from.  Also I wonder how apple mail handles it, maybe they consider anything "multipart/unknown" as "mixed"?  I'll keep this here and see how Thunderbird handles it since that's what I have access to.

Thanks for reporting :)

**Comment by zbateson** [comment]
https://www.w3.org/Protocols/rfc1341/7_2_Multipart.html in 7.2.2:

>  Any multipart subtypes that an implementation does not recognize should be treated as being of subtype "mixed". 

This has been released in 1.2.2.
